letsencrypt/client/display.py

```diff
@@ -74,12 +75,20 @@ def success_installation(self, domains):
             + gen_https_names(domains) + "!", width=self.width)
 
     def display_certs(self, certs):
-        list_choices = [
-            (str(i+1), "%s | %s | %s" %
-             (str(c["cn"].ljust(self.width - 39)),
-              c["not_before"].strftime("%m-%d-%y"),
-              "Installed" if c["installed"] else ""))
-            for i, c in enumerate(certs)]
+        list_choices = []
+        for i, c in enumerate(certs):
+            if c['installed']:
+                if c['installed'] == CONFIG.CERT_DELETE_MSG:
+                    status = "Deleted/Moved"
+                else:
+                    status = "Installed"
+            else:
+                status = ""
+
+            list_choices.append((str(i+1), "{0} | {1} | {2}".format(
+                str(c['cn'].ljust(self.width-39)),
+                c['not_before'].strftime("%m-%d-%y"),
+                status)))
```

Comment:
I decided to completely remove this code from the PR.  It didn't actually implement what I believe to be necessary (it didn't go far enough).  I think the revoker/display class need overhauls, but I decided to leave it to another PR.